src/components/pages/datasets/pep/PepList.jsx

```diff
@@ -98,11 +99,16 @@ const PepList = () => {
           'updated_at',
         ]}
         axiosConfigs={{ useProjectToken: true }}
+        rowLinkUrl={match.url}
+        history={history}
```

Comment:
just add onRowClick event for Table component

Reply:
done